api/v1beta1/tempest_types.go

```diff
@@ -518,6 +520,10 @@ func init() {
 	SchemeBuilder.Register(&Tempest{}, &TempestList{})
 }
 
+func (instance Tempest) IsReady() bool {
+	return instance.Status.Conditions.AllSubConditionIsTrue()
+}
+
```

Comment:
I do not know whether this is just a WIP change but I suggest we replace every usage of `IsReady()` with `instance.Status.Conditions.AllSubConditionIsTrue()`